provider/terraform/property_override.rb

```diff
@@ -0,0 +1,111 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'api/object'
+require 'provider/abstract_core'
+require 'provider/property_override'
+
+module Provider
+  class Terraform < Provider::AbstractCore
+    # Collection of fields allowed in the PropertyOverride section for
+    # Terraform. All fields should be `attr_reader :<property>`
+    module OverrideFields
+      attr_reader :exclude
+      # Adds a DiffSuppressFunc to the schema
+      attr_reader :diff_suppress_func
+      # TODO(rosbo): Consider sharing this functionality with the other tools.
+      attr_reader :default_value
+      # Adds `Sensitive: true` to the schema
+      attr_reader :sensitive
+      # Adds a ValidateFunc to the schema
```

Comment:
How about we write the comments after the reader? Something like:

```
attr_reader :sensitive   # Adds `Sensitive: true` to the schema
```

ditto throughout

Reply:
done